### ISSUE TITLE
Unable to import encrypted JSON exports created using CLI in the web vault UI

### ISSUE THREAD

**Issue by Tarun-Sri-Sai** [bitwarden/clients#12281]
### Steps To Reproduce

1. Go to 'vault.bitwarden.com' and login to your vault.
2. Click on 'Tools' > 'Import Data'.
3. Enter the destination information and choose 'Bitwarden (json)' as the file format for the data.
4. Upload the encrypted JSON export made using Bitwarden CLI's `bw export`.
5. Click 'Import data'.

### Expected Result

Vault data should be imported into the destination.

### Actual Result

I get an 'Import error' in the dialog with empty 'Name' and the 'Description': 'Invalid file password, please use the password you entered when you created the export file.'

In the DevTools console, I get the error `[Encrypt service] MAC comparison failed. Key or payload has changed. Key type AesCbc256_HmacSha256_B64Payload type AesCbc256_HmacSha256_B64 Decrypt context: no context`

### Screenshots or Videos

Dialog after clicking 'Import data':
![Screenshot 2024-12-07 162406](https://github.com/user-attachments/assets/1d127d22-575e-4a51-801e-bc844236c15a)

### Operating System

Windows

### Operating System Version

Windows 10 Home

### Web Browser

Brave

### Browser Version

1.73.97

### Build Version

2024.11.2

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Tarun-Sri-Sai** [comment]
@bwbug, no, I get the same exact error when importing via CLI as well. 

Additionally, I tried two different exports, one using CLI and the other using the web vault. When importing the CLI export I get the issue mentioned above, when importing the web vault export, however, I get: `File contains unassigned items.`

This error also occurs when importing via CLI.

**Comment by Tarun-Sri-Sai** [comment]
There's also around a 2.5KB difference between the CLI export and the web export:

| Export type | Size in bytes |
| - | - |
| CLI export  | 93,449 |
| Web export | 95,945 |

**Comment by Greenderella** [comment]
Hi there,

Could you please write us back using our “Contact support” form located on our Help Center (https://bitwarden.com/help/). This will enable our support team to review your case. You may include a link to this issue in the message content for reference.

Thanks!
